[2024-03-22T04:05:08.869+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task manual__2024-03-22T04:05:07.393572+00:00 [queued]>
[2024-03-22T04:05:08.874+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task manual__2024-03-22T04:05:07.393572+00:00 [queued]>
[2024-03-22T04:05:08.875+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-03-22T04:05:08.883+0000] {taskinstance.py:2217} INFO - Executing <Task(BashOperator): second_task> on 2024-03-22 04:05:07.393572+00:00
[2024-03-22T04:05:08.885+0000] {standard_task_runner.py:60} INFO - Started process 997 to run task
[2024-03-22T04:05:08.893+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'manual__2024-03-22T04:05:07.393572+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp891atmaw']
[2024-03-22T04:05:08.895+0000] {standard_task_runner.py:88} INFO - Job 10: Subtask second_task
[2024-03-22T04:05:08.928+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_task manual__2024-03-22T04:05:07.393572+00:00 [running]> on host 0b9fd85215b3
[2024-03-22T04:05:08.975+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ethan_wu' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-22T04:05:07.393572+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-22T04:05:07.393572+00:00'
[2024-03-22T04:05:08.975+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-22T04:05:08.976+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2024-03-22T04:05:08.982+0000] {subprocess.py:86} INFO - Output:
[2024-03-22T04:05:08.983+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2024-03-22T04:05:08.983+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-22T04:05:08.997+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20240322T040507, start_date=20240322T040508, end_date=20240322T040508
[2024-03-22T04:05:09.027+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-22T04:05:09.041+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
